justfile

```diff
@@ -11,7 +11,7 @@ check:
 
 # Lint everything.
 lint:
-  cargo clippy --workspace --all-targets --all-features -- --deny warnings
+  cargo +nightly clippy --workspace --all-targets --all-features -- --deny warnings
```

Comment:
Would be really great to have single source of truth for things like this but I'm not sure if it's possible.

Reply:
We could have a `contrib/cargo-invocation.sh` script which you could call with "stable" "nightly" "msrv" etc and it would yield cargo invocations.

We would need to manually keep it up to date; maybe could try to tie this into one of our weekly "make a PR" github actions.

But out of scope for this PR :).